tree/tree/src/InternalTreeUtils.cxx

```diff
@@ -165,9 +229,16 @@ RFriendInfo GetFriendInfo(const TTree &tree)
    // internal TTree, if any, will be automatically loaded in each task just like they would be automatically
    // loaded here if we used tree.GetTree()->GetListOfFriends().
    const auto *friends = tree.GetListOfFriends();
-   if (!friends)
+   if (!friends || friends->GetEntries() == 0)
```

Comment:
Indeed, we first check that the `GetListOfFriends` function returns a valid `TList *`. Then, we need to check that the list actually has at least one element, otherwise it means it is just an empty list

Reply:
I guess the question is whether there is any situation in which the `TList` can actually has size 0 (rather than being null).
I don't know the answer so I don't mind the extra check :man_shrugging: